src/rez/cli/rm.py

```diff
@@ -46,6 +52,27 @@ def remove_package(opts, parser):
         sys.exit(1)
 
 
+def remove_package_family(opts, parser):
+    from rez.vendor.version.requirement import VersionedObject
+    from rez.package_remove import remove_package_family
+
+    if opts.dry_run:
+        parser.error("--dry-run is not supported with --family")
+
+    if not opts.PATH:
+        parser.error("Must specify PATH with --family")
+
+    obj = VersionedObject(opts.family)
+    if obj.version:
+        parser.error("Expected package name, not version")
+
+    if remove_package_family(obj.name, opts.PATH):
```

Comment:
Should we try/except and print a nice error message when `remove_package_family` raises a `PackageRepositoryError` error?

Something like:
```python
success = False
try:
    success = remove_package_family(obj.name, opts.PATH)
except PackageRepositoryError as exc:
   print("Failed to remove package family: {0}".format(exc), file=sys.stderr))
   sys.exit(1)

if not success:
    print("Package family not found.", file=sys.stderr)
    sys.exit(1)
```

It would be more user friendly than a traceback.

Reply:
agreed